configs/components/augeas.rb

```diff
@@ -117,6 +123,9 @@
     pkg.environment "LDFLAGS", settings[:ldflags]
   end
 
+  # fix libtool linking on big sur
+  pkg.configure { ["/usr/local/opt/bin/autoreconf --force --install"] } if platform.is_macos?
```

Comment:
When I attempted to build runtime with this PR I ran into an issue with the path here, I had to set it to `/usr/local/bin/autoreconf`.